spec/units/transcoding_spec.rb

```diff
@@ -82,6 +87,28 @@ def generate_special_derivatives
     end
   end
 
+  describe "with an attached RAW image", unless: ENV['TRAVIS'] == 'true' do
```

Comment:
Not sure why the TIFF test was excluded in Travis, but XLS and PPT were excluded because of the missing soffice dependency.  Give it a shot, and if the test bombs, go ahead and exclude it.